packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -1146,14 +1146,28 @@ export default class BackupNg {
         $defer.call(xapi, 'deleteVm', snapshot)
       }
 
+      const compress = getJobCompression(job)
+      const { $pool: pool } = snapshot
```

Comment:
Don't use destructuring for this: it's unnecessary complex and is not even shorter.